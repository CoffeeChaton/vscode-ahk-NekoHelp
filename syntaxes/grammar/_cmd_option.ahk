;https://www.autohotkey.com/docs/v1/lib/Send.htm#keynames

~F11:: foo()

foo() {
    Send, {}}
    GoSub, Label1
}

Label1:
    Send, {{}
Return

