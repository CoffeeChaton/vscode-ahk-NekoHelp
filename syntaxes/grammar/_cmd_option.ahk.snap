>;https://www.autohotkey.com/docs/v1/lib/Send.htm#keynames
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>
>~F11:: foo()
#^ source.ahk meta.hotkey.line.ahk storage.modifier.hotkey.ahk
# ^^^ source.ahk meta.hotkey.line.ahk meta.2.hotkey.line.ahk markup.bold.hotkey.ahk
#    ^^ source.ahk meta.hotkey.line.ahk
#      ^ source.ahk
#       ^^^ source.ahk entity.name.function.ahk
#          ^ source.ahk punctuation.bracket.ahk
#           ^ source.ahk punctuation.bracket.ahk
>
>foo() {
#^^^ source.ahk entity.name.function.ahk
#   ^ source.ahk punctuation.bracket.ahk
#    ^ source.ahk punctuation.bracket.ahk
#     ^ source.ahk
#      ^ source.ahk punctuation.bracket.ahk
>    Send, {}}
#^^^^ source.ahk
#    ^^^^ source.ahk meta.command.line.ahk support.function.ahk
#        ^ source.ahk meta.command.line.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.ahk
#          ^ source.ahk meta.command.line.ahk punctuation.bracket.ahk
#           ^ source.ahk meta.command.line.ahk string.command.option.ahk
#            ^ source.ahk meta.command.line.ahk punctuation.bracket.ahk
>    GoSub, Label1
#^^^^ source.ahk
#    ^^^^^ source.ahk meta.goto.gosub.ahk keyword.control.flow.ahk
#         ^^ source.ahk meta.goto.gosub.ahk
#           ^^^^^^ source.ahk meta.goto.gosub.ahk entity.name.function.label.ahk
>}
#^ source.ahk punctuation.bracket.ahk
>
>Label1:
#^^^^^^^ source.ahk labelline.ahk entity.name.function.label.ahk
>    Send, {{}
#^^^^ source.ahk
#    ^^^^ source.ahk meta.command.line.ahk support.function.ahk
#        ^ source.ahk meta.command.line.ahk punctuation.ahk
#         ^ source.ahk meta.command.line.ahk
#          ^ source.ahk meta.command.line.ahk punctuation.bracket.ahk
#           ^ source.ahk meta.command.line.ahk string.command.option.ahk
#            ^ source.ahk meta.command.line.ahk punctuation.bracket.ahk
>Return
#^^^^^^ source.ahk keyword.control.flow.ahk
>
>